pgactivity/__main__.py

```diff
@@ -1,4 +1,10 @@
+import sys
+
 from .cli import main
 
 if __name__ == "__main__":
-    main()
+    try:
+        main()
+    except KeyboardInterrupt:
+        sys.exit(1)
+
```

Comment:
This change does seem needed, as the change in `cli.main()`, below is sufficient I think.

(This file is used as `python3 -m pgactivity [ARGS]`.)

Reply:
Yes, thank you for pointing that out. I've changed `__main__.py` back to match master.